docs/dev/README.md

```diff
@@ -16,14 +18,14 @@ They may require some small tweaks for your environment (as it will be different
 
 ### Deploying / upgrading Jenkins / plugin
 
-1. Build the plugin (mvn verify)
+1. Build the plugin (`mvn verify`) (from the root of the repository)
 2. build and tag the docker image
    ```
-      docker build .. -f Dockerfile -t jenkins-k8s-creds:latest
+      docker build ../.. -f Dockerfile -t jenkins-k8s-creds:latest

```

Comment:
`:latest` is implicit if you don't supply any tag, so it's redundant to put it in. You can just do `docker build ../../ -F Dockerfile -t jenkins-k8s-creds`

Reply:
fixed